pytube/contrib/search.py

```diff
@@ -42,8 +42,12 @@ def completion_suggestions(self):
         if self._completion_suggestions:
             return self._completion_suggestions
         if self.results:
-            self._completion_suggestions = self._initial_results['refinements']
-        return self._completion_suggestions
+            try:
+                if self._initial_results['refinements']:
```

Comment:
why is there an if here?

Reply:
Hey thanks for pointing this out, this was redundant and hence I've gotten rid of it.